pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -224,14 +224,20 @@ func (b *OFBridge) createGroupWithType(id GroupIDType, groupType ofctrl.GroupTyp
 	return g
 }
 
-func (b *OFBridge) DeleteGroup(id GroupIDType) error {
+// DeleteGroup deletes a specified group in groupDb. If deleteGroupOnOVS sets to true, the group deletion transaction
+// will be synced to OVS. Note that to record OVS operation in Prometheus, openFlowClient.DeleteOFEntries([]binding.OFEntry{groupCache})
+// should be called in advance, and deleteGroupOnOVS is unnecessary.
+func (b *OFBridge) DeleteGroup(id GroupIDType, deleteGroupOnOVS bool) error {
```

Comment:
I think the methods is a little confusing as `CreateGroup` only construct group in memory while `DeleteGroup` could delete the group in OVS. Methods should be symmetrical so clients can use similar methods to manage the same resource.
Since there isn't even an actual case needing `deleteGroupOnOVS` to be true, I think we should just make CreateGroup and DeleteGroup symmetrical.

Reply:
updated